### PR TITLE
Works and about feature

### PR DESCRIPTION
**Pull request by JbirdL86**
In this branch we have added more content (2 sections), this content was added mainly using flexbox and grids. Also, we've added some images and a font file .ttf in order to get a type of font that wasn't available to link from the internet. The main changes are visible in the following files and folders:

- index.html
- style.css
- assets/

I hope you like my project so far! :zap: :+1:

### CODE REVIEW

**Review by ElijahTobs — COMMENTED** [review]
# STATUS: INVALID 🔴 
## Hi @JbirdL86  👋 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has **failing linter checks**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -16,40 +16,204 @@
...
+                       text ever since the 1500s, when an unknown printer took a 
+                       standard dummy text.
+                    </p>
+                    <div class="button-grid">
+                        <div class="textgrid button1">css</div>
+                        <div class="textgrid button2">html</div>
+                        <div class="textgrid button3">bootstrap</div>
+                        <div class="textgrid button4">Ruby</div>
+                        <div class="seeproject">See Project</div>
+                    </div>
```
Comment (ElijahTobs) [inline index.html]: - [ ] The list of project technologies (HTML, CSS, Bootstrap and Ruby) should use a `ul` for grouping and each of them a `li`. All of such occurrences should be fixed.
- [ ] The button `See project` should be a link `a` or `button` tag (Kindly fix all other occurrences).

File: index.html [hunk 2]
```diff
@@ -16,40 +16,204 @@
...
+                    </article>
+                </section>
+            </section>
+            <section class="aboutme-container">
+                <section class="aboutme-profile">
+                    <div>
+                        <h1 class="aboutme">About me</h1>
+                        <p class="context">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me..</p>
+                    </div>
+                    <div class="resume">Get My Resume</div>
```
Comment (ElijahTobs) [inline index.html]: - [ ] `Get My Resume` should be a link `a` or `button` tag (Kindly fix all other occurrences).

File: index.html [hunk 3]
```diff
@@ -16,40 +16,204 @@
...
+                            <a>Codepen</a>
+                        </li>
+                        <li>
+                            <a>Gitlab</a>
+                        </li>
+                        <li>
+                            <a>Terminal</a>
+                        </li>
+                    </ul>
+                </section>
```
Comment (ElijahTobs) [inline index.html]: - [ ] The technology categories ("Languages", "Frameworks" and "Skills") is a list itself and should be grouped with the `ul` tag, and each of them in a `li`
- [ ] The technology in each category ("JavasScript", "HTML" and "CSS") should also be grouped using the `ul` tag and each of them a `li`. See an illustration below
```html
<ul class="categories">
    <li>
        <p>Languages</p>
        <ul class="technology">
            <li>JavaScript</li>
            <li>Ruby</li>
            <li>HTML</li>
        </ul>
    </li>
    <li>
        <p>Languages</p>
        <ul class="technology">
            <li>JavaScript</li>
            <li>Ruby</li>
            <li>HTML</li>
        </ul>
    </li>
    <li>
        <p>Languages</p>
        <ul class="technology">
            <li>JavaScript</li>
            <li>Ruby</li>
            <li>HTML</li>
        </ul>
    </li>
</ul>
```

File: style.css [hunk 4]
```diff
@@ -92,3 +107,251 @@ header {
...
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  background: #ff6b00;
+  padding: 12px;
+}
```
Comment (ElijahTobs) [inline style.css:312]: - [ ] You should add the button styling according to the figma design: hover, pressed and active. Do this for all the buttons

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @JbirdL86  👋

You've done really well so far! 👍 

Your README is professional and you've followed the Github flow rules. ✔️ 
Good Pull Request (PR) title and summary ✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️

_See the inline comments below_

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

- It is advisable that your commit messages be capitalized, and written in the imperative tense. You can watch [this video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6) on how to edit your commit messages using `git rebase`.
Examples of a good commit message:
 i. Add HTML files to github
ii. Add CSS files to github

Check for changes marked as Optional in the inline comments below.


#### If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

### Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#), however, if you feel this review is unfair, you can request a second opinion using this [form](https://airtable.com/shrQAqnBwek5a0O0s).

## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

File: index.html (hunk 1 above)
Comment (DeliceLydia) [reply index.html]: The issue was well addressed. Good job!✅

File: index.html (hunk 2 above)
Comment (DeliceLydia) [reply index.html]: Well done get my resume is now a button!✅

File: index.html (hunk 3 above)
Comment (DeliceLydia) [reply index.html]: Resolved this as well ✅

File: style.css (hunk 4 above)
Comment (DeliceLydia) [reply style.css:312]: Well resolved this as well✅

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved ✨✨🎉 

Hi @JbirdL86, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**